pyam_analysis/core.py

```diff
@@ -536,6 +537,30 @@ def metadata(self, meta=None, name=None, filters={},
                 meta = meta[keep_col_match(meta[col], values)]
             return return_df(meta, display, idx_cols)
 
+    def to_excel(self, excel_writer, filters={}, exclude_cat=[],
+                 sheet_name='data', index=False, **kwargs):
+        """Write timeseries data to Excel using the IAMC template convention
+        (wrapper for `pandas.DataFrame.to_excel()`)
+
+        Parameters
+        ----------
+        excel_writer: string or ExcelWriter object
+             file path or existing ExcelWriter
+        filters: dict, optional
+            filter by model, scenario, region, variable, level, year, category
+            see function _select() for details
+        exclude_cat: None or list of strings, default []
+            exclude all scenarios from the listed categories
+        sheet_name: string, default 'data'
+            name of the sheet that will contain the (filtered) IamDataFrame
+        index: boolean, default False
+            write row names (index)
+        """
+        df = self.timeseries(filters, exclude_cat).reset_index()
+        df.columns = [str.title(i) if type(i) == str else i
```

Comment:
see previous notes on investigating a `capitalize_columns()` function (which may not be necessary if we want to expose `df.rename()`.